charts/ccd-case-activity-api/values.yaml

```diff
@@ -16,6 +16,8 @@ nodejs:
   environment:
```

Comment:
I added new things in the values. MORE DOUCH . I might need to increase the version of chart  with the one proposed by from develop.